test/extended/router/certs.go

```diff
@@ -65,6 +111,15 @@ pgfj+yGLmkUw8JwgGH6xCUbHO+WBUFSlPf+Y50fJeO+OrjqPXAVKeSV3ZCwWjKT4
 u3YLAbyW/lHhOCiZu2iAI8AbmXem9lW6Tr7p/97s0w==
 -----END RSA PRIVATE KEY-----
 `
+)
+
+var _ = g.Describe("[sig-network][Feature:Router][apigroup:route.openshift.io]", func() {
```

Comment:
I think this string should have "[apigroup:template.openshift.io]" since the `BeforeEach` block applies the `router-common.yaml` fixture (which contains a template) on line 153, right?  But that's a discrepancy that predates this commit and that is resolved by a follow-up commit, so it doesn't really matter.